content/en/cloud_cost_management/multisource_querying/_index.md

```diff
@@ -99,6 +96,7 @@ The following FOCUS tags are available in Cloud Cost Management:
 | `servicename` | An offering that can be purchased from a provider (for example, cloud virtual machine, SaaS database, professional services from a systems integrator). |
 | `billingaccountid` | The identifier assigned to a billing account by the provider. |
 | `billingaccountname` | The display name assigned to a billing account. |
+| `billingcurrency` | The currency in which a cloud bill was ingested in. |
```

Comment:
I dont think this is exactly right 
https://focus.finops.org/focus-specification/v1-1/#:~:text=2.5.-,Billing%20Currency,-Billing%20currency%0Ais

It is more that this is the currency the customer paid their bill in. Ingestion is not really a word I would use in this context on a public doc personally. It is more a technical term